staff/juan-sensio/cv/index.html

```diff
@@ -4,39 +4,65 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <link rel="stylesheet" href="styles.css">
-    <link href="https://fonts.googleapis.com/css?family=Montserrat" rel="stylesheet">
     <title>CV</title>
 </head>
 <body>
-    <h1>Aitor Tilla</h1>
+    <header>
+        <nav id="top">
+            <a href="#datos">Datos Personales</a>
+            <a href="#educacion">Educación</a>
+            <a href="#experiencia">Experiencia</a>
+            <a href="#habilidades">Habilidades</a>
+        </nav>
+    </header>
 
-    <div>
-        <h2>Datos Personales</h2>
-        <p>Carrer de Roc Boronat, 35 (Barcelona)</p>
-        <p>698203948</p>
-        <p>aitortilla@gmail.com</p>
-    </div>
-
-    <div>
-        <h2>Educación</h2>
-        <h3>Skylab coders (www.skylabcoders.com). </h3>
-        <p>Full-Stack Web Development Bootcamp.</p>
-        <h3>La Universidad de la Calle (UDC).</h3>
-        <p>No tengo la E.S.O, pero mi abuela dice que soy muy espabilado.</p>
-    </div>
-    
-    <div>
-        <h2>Experiencia</h2>
-        <h3>NASA</h3>
-        <p>Desarrollo del software de aviónica del programa Space Shuttle.</p>
-    </div>
+    <main>
+        <h1>Aitor Tilla</h1>
+        
+        <section id="datos">
+            <h2>Datos Personales</h2>
+            <p>Carrer de Roc Boronat, 35 (Barcelona)</p>
+            <p>698203948</p>
+            <p><mark>aitortilla@gmail.com</mark></p>
+        </section>
     
-    <div>
-        <h2>Habilidades</h2>
-        <p>Git nivel avanzado (pull request legend).</p>
-        <p>Beber chupitos con el codo.</p>
-    </div>
+        <section id="educacion">
+            <h2>Educación</h2>
+            <section>
+                <h3>Skylab coders (www.skylabcoders.com). </h3>
+                <p><mark>Full-Stack</mark> Web Development Bootcamp.</p>
+                <time>2018</time>
+            </section>
+            <section>
+                <h3>La Universidad de la Calle (UDC).</h3>
+                <p>No tengo la E.S.O, pero mi abuela dice que soy muy espabilado.</p>
+                <time>1990-2018</time>
+            </section>
+        </section>
+        
+        <section id="experiencia">
+            <h2>Experiencia</h2>
+            <section>
+                <h3>NASA</h3>
+                <p><mark>Desarrollo del software </mark>de aviónica del programa Space Shuttle.</p>
+                <time>2013-2015</time>
+            </section>
+        </section>
+        
+        <section id="habilidades">
+            <h2>Habilidades</h2>
+            <p><mark>Git</mark> nivel avanzado (pull request legend).</p>
+            <p><mark>Git</mark> nivel avanzado (pull request legend).</p>
+            <p>Beber chupitos con el codo.</p>
+        </section>
+    </main>
+
+    <footer>
+        <p>Last update: <time>2018-04-06</time>.</p>
+        <nav>
```

Comment:
Nav is more useful for multiple links. I'd remove it and leave the anchor alone.

Reply:
@alexgbor change implemented.